plugin-server/package.json

```diff
@@ -94,7 +94,8 @@
         "tldts": "^6.1.57",
         "uuid": "^9.0.1",
         "v8-profiler-next": "^1.9.0",
-        "vm2": "3.9.18"
+        "vm2": "3.9.18",
+        "siphash": "git://github.com/robbie-c/siphash-js.git#34055d9"
```

Comment:
It might make sense to put it under the posthog org if you don't have plans to use it in any other project of yours. Could even be called `siphash-ts` now that there are bindings 🤣. Forking it was a good idea, though, I prefer that to vendoring it

Reply:
This is probably the right solution, I'm just trying to avoid the headache of getting permissions to do stuff 😅

Reply:
Done!